src/main/java/frc/robot/Robot.java

```diff
@@ -26,7 +26,7 @@
  * project.
  */
 public class Robot extends LoggedRobot {
-  public static final boolean isCompetition = false;
```

Comment:
Should this be true or false on main?

Reply:
If we continue to plan on having a separate branch for comp changes, then I would say keep it false.